vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -139,6 +141,22 @@ public void initUiAndData() {
 
         setTitle(R.string.tchap_room_creation_title);
 
+        // Initialize the room messages retention period which is underlined and clickable
+        setRoomRetentionPeriod(RoomRetentionKt.DEFAULT_RETENTION_VALUE);
+        roomMessageRetentionText.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                new RoomRetentionPeriodPickerDialogFragment(TchapRoomCreationActivity.this)
+                        .create(mRetentionPeriod, (number) -> {setRoomRetentionPeriod(number); return null;})
+                        .show();
+
+            }
+        });
```

Comment:
It's a shame to do all that if ENABLE_ROOM_RETENTION is false :)

Reply:
yes, but this is temporary (I will ignore this remark)